extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -20,6 +20,23 @@ import {user} from '../../../src/log';
 import {removeElement} from '../../../src/dom';
 import {timer} from '../../../src/timer';
 import {toggle} from '../../../src/style';
+import {waitForRenderStart} from '../../../3p/integration';
+
+
+/**
+ * Allows listening for message from the element, unlisten afterward.
+ *
+ * @param {!Element}.
+ * @param {string} typeOfMessage.
+ * @param {function(!Object, !Window, string)} callback Called when a message of
+ *     this type arrives for this iframe.
+ */
+function listenOnce(element, typeOfMessage, callback) {
+  element.addEventListener(typeOfMessage, () => {
+    callback();
```

Comment:
This'll throw an error on the second message. We have a helper for this, too, called [`listenOnce`](https://github.com/ampproject/amphtml/blob/fe63400cb62cd132ae279357e6d546fbd9d0010c/src/event-helper.js#L42-L68).